CONTRIBUTING.md

```diff
@@ -216,5 +225,5 @@ message automatically.
 
 Forgot to add DCO to a commit? Amend it with `git commit --amend -s`.
 
-[^1]: Performance improvements in non-hot paths are unlikely to be considered
-    valuable.
+\[^1\]: Performance improvements in non-hot paths are unlikely to be considered
```

Comment:
Mh, this mdformat broke this footnote. I think you also need to install the extensions that are installed in the docker container for it to correctly deal with github-style footnotes.

Reply:
@roypat thank you again for the review!
This one comment of yours made me realize that a better way forward would be to actually make use of what existed so I dove a bit more and realized that `./tools/devctr` is the thing keeping track of the linting deps.
Since that project is a poetry project, I figured out a way to actually make use of it (and included pyenv because python 3.10 is somewhat old).

Thank you for catching this!